vault/resource_auth_backend_test.go

```diff
@@ -214,7 +214,9 @@ func TestResourceAuthTune(t *testing.T) {
 			{
 				Config: testResourceAuthTune_initialConfig(backend),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAuthMountExists(resName, &resAuthFirst),
+					testutil.TestAccCheckAuthMountExists(resName,
+						&resAuthFirst,
+						testProvider.Meta().(*provider.ProviderMeta).GetClient()),
```

Comment:
This is currently erroring since `GetClient` returns two values instead of one. What we want to use is the following instead:
```suggestion
						testProvider.Meta().(*provider.ProviderMeta).MustGetClient()),
```

Reply:
Oh whoops, yes it does. Thanks for catching that:  Updated all the introduced `*.GetClient()` calls to be `*.MustGetClient()`.